jsapp/js/envStore.ts

```diff
@@ -115,20 +112,19 @@ class EnvStore extends Reflux.Store {
     }
 
     this.isReady = true;
-    this.trigger(this.data);
   }
 
-  getLanguages() {
+  get languages() {
```

Comment:
Interesting rename. I didn't see any usage for `getLanguages(` or `languages(`, internally or externally -- other class functions here are just using `this.data.all_languages` directly.

Reply:
I think that store was built with all different useful methods that some didn't find any use. I think it would make sense to update all `this.data.all_languages` usages to `this.languages`, or to just remove this method :P Especially that `feature/nlp` will remove `all_languages` from here :)

Reply:
Removing this makes the most sense to me 👍

Reply:
Made this change over in #4030